src/lib/ppx_coda/versioned.ml

```diff
@@ -93,7 +93,7 @@ let validate_rpc_type_decl inner3_modules type_decl =
 
 let validate_plain_type_decl inner3_modules type_decl =
   match inner3_modules with
-  | ["T"; module_version; "Stable"] ->
+  | ["T"; module_version; "Stable"] | module_version :: "Stable" :: _ ->
```

Comment:
maybe a comment here to say the `T` variant can be removed, once the `Registration` functors go away.